lib/blocs/chat_bloc.dart

```diff
@@ -20,10 +20,21 @@ class ChatBloc {
   static StreamSubscription<NewMessageEvent>? _messageSubscription;
 
   final RxList<Chat> _chats = <Chat>[].obs;
-  RxList<Chat> get chats => _chats;
+
+  RxList<Chat> get chats => _chats
+      .where((e) => SettingsManager().settings.skin.value == Skins.iOS ? !(e.isPinned ?? false) : true)
+      .toList()
+      .obs;
+
+  RxList<Chat> get pinnedChats => _chats
+      .where((e) => SettingsManager().settings.skin.value == Skins.iOS ? e.isPinned ?? false : false)
+      .toList()
+      .obs;
```

Comment:
For this I'd prefer if the chat list returned all the chats so that we can use this getter everywhere else besides the conversation list to get all the chats, regardless of pinned status. If you could maybe make another helper function like I did at the bottom I think that would be good, something like this:

```
pinnedHelper(bool pinned) {
    if (pinned)
      return this.where((e) => e.isPinned ?? false).toList();
    else
      return this;
  }

Reply:
You could use that in conjunction with the archived helper to get pinned & archived chats, or make another helper for that specific case.

Reply:
The reason I did it this way was that there were no uses of this chats getter outside of conversation_list lol. Will make this change tho

Reply:
> The reason I did it this way was that there were no uses of this chats getter outside of conversation_list lol. Will make this change tho

Yea in the current state it's like that but once we start consolidating all of the blocs / managers we may end up using it elsewhere since we want everything to be reactive and update state as things change. Shouldve been more clear ab that

Reply:
Done.